magento/2.4/artifakt_scripts/composer_install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ -z "$ARTIFAKT_COMPOSER_VERSION" ];then ARTIFAKT_COMPOSER_VERSION="2.3.7"; fi 
```

Comment:
FYI In Bash you can use the syntax `${ARTIFAKT_COMPOSER_VERSION:-"2.3.7"}` instead of this to do _use variable or default value if undefined_